tools/cli/go-whisk/whisk/client.go

```diff
@@ -267,12 +279,13 @@ func (c *Client) Do(req *http.Request, v interface{}, ExitWithErrorOnTimeout boo
         if req.Body != nil {
             fmt.Println("Req Body")
             if !IsDebug() {
-                if truncatedBody, req.Body, err = bodyTruncator(req.Body); err != nil {
+                if truncatedBody, req.Body, err = BodyTruncator(req.Body); err != nil {
                     return nil, err
                 }
-                fmt.Println(truncatedBody)
+                fmt.Println(ObfuscateText(truncatedBody, obfuscateArr))
             } else {
-                fmt.Println(req.Body)
+                data, _ := ioutil.ReadAll(req.Body)
```

Comment:
Does the `Debug()` all below still work with this change?

Reply:
Good point, looks like the debug statement is now an empty JSON string.  Let me fix that.

Reply:
So the interesting thing i found here, is that the Debug (and my code) both flush the request body, so whenever you either pass in verbose or debug, they flush the body and so send a request with no body, which causes the command to fail.  Going to fix this.  Just thought I'd put this in here since the issue is bigger than initially expected.